src/Microsoft.Azure.CosmosEventSourcing/Aggregates/IAggregateRoot.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Azure.CosmosEventSourcing.Events;
+
+namespace Microsoft.Azure.CosmosEventSourcing.Aggregates;
+
+/// <summary>
+///
```

Comment:
More missing comments... For public APIs we should really try to have the triple slash comments serve as a the docs.

Reply:
I have updated these.